packages/utah-design-system/src/components/Sherlock.stories.jsx

```diff
@@ -87,3 +87,19 @@ export const HasMultiProvider = {
     />
   ),
 };
+
+export const HasError = {
+  render: (args) => (
+    <Component
+      {...args}
+      label="Select a road"
+      placeholder="Search the roads map service"
+      provider={featureServiceProvider(
```

Comment:
A tip for the future, the mock service worker is great for network related things like this